scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController_SpecificSchedule_Test.java

```diff
@@ -105,6 +115,50 @@ public void testCreateAndGetSchedules() throws Exception {
 
 		assertCreateAndGetSchedules(multipleAppIds, 1);
 		assertCreateAndGetSchedules(multipleAppIds, 5);
+
+	}
+
+	@Test
+	@Transactional
+	public void testCreateSchedulesMultipleConcurrent() throws Exception {
+		ExecutorService service = Executors.newFixedThreadPool(20);
```

Comment:
For now, we remove this method test case. And once we find a better way ,we will add it through a  separate story.